### DISCUSSION TITLE
Opening Settings Failed - MacOS

### DISCUSSION THREAD

**Discussion by kcoop050** [SafeExamBrowser/seb-mac#147] [Q&A]
Hello,
I have a student who is using MacOS High Sierra and is experiencing issues when trying to launch SEB in Moodle. I have gone through the steps with her one by one and when she tries to launch SEB via Moodle and login using Single Sign-On, it presents a Mac error "opening settings failed". 
I have used my Mac and was able to take the exam as a student with no issues. 
What advice can I provide to this student? Their exam is in 3 days. 
Thanks so much

<img width="512" alt="Screen Shot 2021-10-28 at 10 16 11 pm" src="https://user-images.githubusercontent.com/93326475/139253469-27d79a1a-363c-4171-ac05-3d6490d0809d.png">

**Comment by danschlet** [comment]
Please ALWAYS indicate which SEB version was used. Otherwise we cannot help at all, unfortunately.

**Reply by kcoop050** [reply]
Hi Daniel. Thank you so much for getting back to me. The student was using 3.0.

**Comment by danschlet** [comment]
If it wasn't the current one, 3.0, then she should definitely update to the latest SEB version. If it already is 3.0, it would be helpful for us to know if it works correctly with the previous version 2.3.2 (the [download link](https://sourceforge.net/projects/seb/files/seb-macosx/seb_macos_2.3.2/SafeExamBrowser-2.3.2.dmg/download) is still available under former releases). 

Best would be, if she could update her macOS version. macOS 10.13 High Sierra is no longer receiving security updates from Apple (which now also will apply to macOS 10.14), so it would be better to update to at least Catalina 10.15.

**Reply by kcoop050** [reply]
I have asked the student to download the legacy version and try to run it again and let me know if it works - I'll come back to you with that. I told her if it doesn't work, then to update her macOS to Catalina 10.15 and give it another try. You have no idea how much I appreciate your reply. Thank you again.
